test/e2e/kuma_deploy_global_remote_test.go

```diff
@@ -104,6 +107,9 @@ metadata:
 
 		err = c2.DeleteKuma(optsRemote...)
 		Expect(err).ToNot(HaveOccurred())
+
+		// restore the original namespace
+		KumaNamespace = originalKumaNamespace
```

Comment:
please put this in defer of after each. If there is a problem with deleting Kuma or Namespace or whatever, it won't restore this value. It would be perfect to pass this as an argument to deploy and rely on this instead of modifying global variable, but this might be difficult without putting Kuma into `deployments` abstraction.